lapce-app/src/command.rs

```diff
@@ -31,7 +31,7 @@ pub struct LapceCommand {
     pub data: Option<Value>,
 }
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
I've changed `CommandKind` and all its subcommand types to implement `Copy`. I'm wary of making a far reaching change like this, but couldn't see any reason why these types shouldn't be `Copy`, so decided to make the change and let the maintainers kick it back if it is incorrect.

There are some clippy warnings about cloning `Copy` types as a result of this change - I'll make another commit fixing these if the change is OK.

Reply:
maybe it can't be `Copy` because of strum? I'm not sure. Let's get the palette one merged in first.

Reply:
I just did a quick search and didn't find anything on `strum` causing problems with `Copy`, and it compiles fine with the change and seems to work - but fair enough. I'll revert the addition of `Copy`.